### DISCUSSION TITLE
The custom domain is already taken

### DISCUSSION THREAD

**Discussion by ShahzaibZulfiqar** [community/community#30660] [Other Feature Feedback, Questions, & Ideas]
Hi I just joined GitHub and I want to add custom domain to page but it is already taken, I verified it but domain shows previous owner GitHub page 
domain is shahzaibali.me
https://github.com/bc170200229/bc170200229.github.io
old owner
https://github.com/Shahzaib-1/Shahzaib-1.github.io

**Comment by jfullstackdev** [comment]
seems it's your other account

![image](https://user-images.githubusercontent.com/47092464/186658676-bde0b223-4f0d-4806-bab8-eb857d5e1176.png)

why he/she would know your other username on GitHub ? that's hard to be guessed unless it's you or you know each other personally
